Warning in <TFile::Init>: no StreamerInfo found in /lustrehome/bdanzi/TestBeam2022/analysistestbeam2022/analysis_2021/drifttubes_offline_analysis/testbeam_analysis/../dchdatareade_test_beam/WaveData/lib/WaveDataDict_rdict.pcm therefore preventing schema evolution when reading this file.
Error in <TFile::TFile>: file ./run_41.root does not exist
Read file with name: ./run_41.root

 *** Break *** segmentation violation



===========================================================
There was a crash.
This is the entire stack trace of all threads:
===========================================================
#0  0x00007f082b8f960c in waitpid () from /lib64/libc.so.6
#1  0x00007f082b876f62 in do_system () from /lib64/libc.so.6
#2  0x00007f082dbb608d in TUnixSystem::StackTrace() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc700/lcg/root/6.14.09-pafccj6//lib/libCore.so
#3  0x00007f082dbb8934 in TUnixSystem::DispatchSignals(ESignals) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc700/lcg/root/6.14.09-pafccj6//lib/libCore.so
#4  <signal handler called>
#5  0x000000000041c9f3 in main ()
===========================================================


The lines below might hint at the cause of the crash.
You may get help by asking at the ROOT forum http://root.cern.ch/forum
Only if you are really convinced it is a bug in ROOT then please submit a
report at http://root.cern.ch/bugs Please post the ENTIRE stack trace
from above as an attachment in addition to anything else
that might help us fixing this issue.
===========================================================
#5  0x000000000041c9f3 in main ()
===========================================================


